tests/tests.js

```diff
@@ -693,4 +698,29 @@ exports.defineManualTests = function (contentEl, createActionButton) {
             });
         });
     }, 'openHardwareBackDefaultAfterNo');
+
+    // Customscheme
+    createActionButton('customscheme', function () {
+        var ref = cordova.InAppBrowser.open('about:blank', '_blank', 'hidden=yes');
+        var openedCustomscheme = false;
+        ref.addEventListener('loadstop', function (e) {
+            // Avoid showing the alert twice on iOS, since loadstop is also being called after the customscheme event.
+            if (!openedCustomscheme) {
+                openedCustomscheme = true;
+                ref.executeScript({ code: 'window.location.replace("custom://test");' });
+            }
+        });
+        ref.addEventListener('customscheme', function (e) {
+            if (e && e.url === "custom://test") {
```

Comment:
I just discovered today that `npm run eslint` has a few failures:

```
/Users/brodybits/Documents/cordova/cordova-plugin-inappbrowser/tests/tests.js
  714:32  error  Strings must use singlequote                    quotes
  715:1   error  Expected indentation of 16 spaces but found 14  indent
  715:15  error  'alert' is not defined                          no-undef
  717:1   error  Expected indentation of 16 spaces but found 14  indent
  717:15  error  'alert' is not defined                          no-undef
  722:13  error  'alert' is not defined                          no-undef

✖ 6 problems (6 errors, 0 warnings)
  3 errors, 0 warnings potentially fixable with the `--fix` option.
```

I can think of 2 possible solutions for `'alert' is not defined`:
- use `// eslint-disable-line no-undef`
- preferred: use `window.alert` (less ugly, more likely to work on Windows platform)

A nice bonus would be to fix `alert` --> `window.alert` throughout `tests.js`.

Reply:
Thanks! I went for the bonus in 93077ef (hoping it will help motivate you to complete the review ;) ). Tested it still works (Android).